CHANGELOG.md

```diff
@@ -7,6 +7,22 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Breaking changes
+
+- Service switched to using p2wsh address format. (#123)
+  Now anchoring service uses segwit addresses in
+  [p2wsh](https://github.com/bitcoin/bips/blob/master/bip-0141.mediawiki#P2WSH) form.
+  This change increases the limit on the number of validators,
```

Comment:
`This change increases the limit number of validators` ?

Reply:
Yes. Up to 20

Reply:
I just meant this expression is not quite correct 😄